### PR TITLE
Datastore network policy support

### PR DESCRIPTION
**Pull request by BorisMikhailovski**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -67,3 +67,4 @@ override.tf.json
 .terraformrc
 terraform.rc
 terraform
+/local/
```
Comment (otlg) [inline .gitignore:70]: @shalomtuby do you need this?

File: satori/resources/resource_datastore_network_policy.go [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+	"encoding/json"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/satoricyber/terraform-provider-satori/satori/api"
+)
+
+func GetNetworkPolicyDefinition() *schema.Schema {
+	return &schema.Schema{
+		Type:        schema.TypeList,
+		Optional:    true,
+		Description: "a Network Policy for a Data Store",
```
Comment (otlg) [inline satori/resources/resource_datastore_network_policy.go:13]: a->A

File: satori/resources/resource_datastore_network_policy.go [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+			},
+		},
+	}
+}
+
+func GetIpRangesSchemaDefinitions() *schema.Schema {
+	return &schema.Schema{
+		Type:        schema.TypeList,
+		Optional:    true,
+		Description: "enable access control from specified IP ranges",
```
[the remark this answers is not in the document]
Comment (BorisMikhailovski) [reply satori/resources/resource_datastore_network_policy.go:55]: Defines IP addresses or CIDR ranges allowed to access the datastore

File: .gitignore (hunk 1 above)
Comment (BorisMikhailovski) [reply .gitignore]: deleted

**Pull request merged**